packages/calcite-components/src/components/stepper/stepper.scss

```diff
@@ -1,3 +1,15 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-stepper-bar-gap: When `horizontal-single`, specifies the space between items in the step bar.
```

Comment:
This also applies to `row-gap` for horizontal layout & `margin-block-end` for vertical layout, me thinks.